### ISSUE TITLE
[Request] ELI5 documentation on swapping out database connection settings.

### ISSUE THREAD

**Issue by Joseph-R** [Netflix/genie#179]
Hey guys,

In the setup docs there is a segment on [modifying database connection settings](https://github.com/Netflix/genie/wiki/Setup#modify-database-connection-settings-optional), but the documentation is pretty sparse.  I was hoping to get some help setting up a connection to a local Postgres database.

**Disclaimer:** We're a Ruby shop, with minimal knowledge of Java frameworks or tools.  And this is where I think we're getting a little lost.  In Genie v1 we hacked some settings in the persistence-sql.xml file.  In attempting to deploy Genie v2, we're getting stuck in the `genie-jpa.xml`.

In any case, if I set `CATALINA_OPTS` to `"-Dspring.profiles.active=prod -Darchaius.deployment.applicationId=genie -Darchaius.deployment.environment=prod"`, then I believe the relevant block from `genie-jpa.xml` is this one:

```
    <beans profile="prod">
        <bean id="dataSource" class="org.apache.commons.dbcp2.BasicDataSource" destroy-method="close">
            <property name="driverClassName" value="com.mysql.jdbc.Driver"/>
            <property name="url" value="jdbc:mysql://127.0.0.1/genie"/>
            <property name="username" value="root"/>
            <property name="password" value=""/>
            <property name="minIdle" value="5"/>
            <property name="maxIdle" value="20"/>
            <property name="maxTotal" value="40"/>
            <property name="validationQuery" value="select 0;"/>
            <property name="testOnBorrow" value="true"/>
            <property name="testOnCreate" value="true"/>
            <property name="testWhileIdle" value="true"/>
            <property name="testOnReturn" value="true"/>
            <property name="minEvictableIdleTimeMillis" value="60000"/>
            <property name="timeBetweenEvictionRunsMillis" value="10000"/>
        </bean>
    </beans>
```

Changing the username and password seems fairly straightforward.  But I could really use an example on how to swap out these first few lines in order to properly load a PostgreSQL driver in Beans:

```
        <bean id="dataSource" class="org.apache.commons.dbcp2.BasicDataSource" destroy-method="close">
            <property name="driverClassName" value="com.mysql.jdbc.Driver"/>
            <property name="url" value="jdbc:mysql://127.0.0.1/genie"/>
```

We considered just going with MySQL, but everything else we run here is Postgres.  If we can easily sub in connection drivers, that would be the ideal solution.  

I'm not sure how many other non-Java teams are attempting to build or deploy Genie, but I think throwing an extra example into the docs would be great for anybody else facing the same issue.

~J

**Comment by tgianos** [comment]
@Joseph-R I believe we got you through this in the google group right?

For 3.0 we're planning on having a lot more documentation. Also since it leverages Spring Boot documentation should be easy to find.

You might also find #196 of interest to you. We're not Postgres experts or DBA's by any means so any feedback/testing help on the scripts would be appreciated.

**Comment by tgianos** [comment]
Part of #154

**Comment by tgianos** [comment]
Covered in Reference Documentation available on website after every build.
